cohort-parent/pom.xml

```diff
@@ -910,6 +1020,24 @@
 							<outputDirectory>${project.build.directory}/unittest_home</outputDirectory>
 						</configuration>
 					</execution>
+					
+					<execution>
+            			<goals>
+            				<goal>analyze-only</goal>
+            			</goals>
+            			<configuration>
+							<failOnWarning>true</failOnWarning>
+							<ignoredUsedUndeclaredDependencies>
+								<!-- Excluding this because the JAXB classes are provided by the JRE as of Java 8, but there are potentially other classes in jakarta that we don't want to wholesale exclude -->
+								jakarta.xml.bind:jakarta.xml.bind-api
+							</ignoredUsedUndeclaredDependencies>
+							
+							<ignoredUnusedDeclaredDependencies> 
+								<ignoredUnusedDeclaredDependency>ca.uhn.hapi.fhir:hapi-fhir-structures-r4</ignoredUnusedDeclaredDependency>	
+								<ignoredUnusedDeclaredDependency>org.slf4j:slf4j-simple</ignoredUnusedDeclaredDependency>			
+							</ignoredUnusedDeclaredDependencies>
```

Comment:
Do we still need this ignoreUnusedDeclaredDependency after you added runtime to slf4j-simple dependency in cohort-cli/pom.xml?

Reply:
Yup, the plugin isn't intelligent enough to realize it's actually used at runtime.